Microsoft.Research/RegressionTest/ClousotTests/v.cs

```diff
@@ -48,6 +50,1516 @@ public TestContext TestContext
+  contractReferenceAssemblies: true"
+  exe: false"
+     ),
+  new Options(
+  sourceFile: @"Microsoft.Research\RegressionTest\Inference\Preconditions-BlogExample.cs"
+  compilerOptions: "/optimize"
+  libPaths: new string[0],
+  references: new[] { "System.Core.dll"
+  options: "-infer autopropertiesensures -suggest requires -includesuggestionsinregression -throwArgExceptionAsAssert -premode backwards -suggest requires -infer requires -nonnull -bounds -show=!! -show progress -show errors -suggest assumes -suggest codefixes -maskverifiedrepairs=false  -missingPublicRequiresAreErrors  -suggest calleeassumes -suggest assumes -suggest requires -suggest necessaryensures -suggest readonlyfields  -infer requires -infer methodensures"
+  contractReferenceAssemblies: true"
+  exe: false"
+     ),
+
+  new Options(
+    sourceFile: @"Microsoft.Research\RegressionTest\Inference\SimplePreconditionPropagation.cs"
+    compilerOptions: "/optimize"
+    libPaths: new string[0],
+    references: new[] { "System.Core.dll"
+    options: "-infer autopropertiesensures -suggest requires -includesuggestionsinregression -throwArgExceptionAsAssert -premode combined -suggest requires -infer requires -nonnull -bounds -show validations -show unreached -show progress "
+    contractReferenceAssemblies: true"
+    exe: false"
+     ),
+  new Options(
+    sourceFile: @"Microsoft.Research\RegressionTest\Inference\RedundantAssumptions.cs"
+    compilerOptions: "/optimize"
+    libPaths: new string[0],
+    references: new[] { "System.Core.dll"
+    options: "-infer autopropertiesensures -suggest requires -includesuggestionsinregression -premode allPaths -suggest requires -nonnull -bounds -show validations -show unreached -show progress -check assumptions -infer methodensures"
+    contractReferenceAssemblies: true"
+    exe: false"
+     ),
+  new Options(
+    sourceFile: @"Microsoft.Research\RegressionTest\Inference\AssertToContracts.cs"
+    compilerOptions: "/optimize"
+    libPaths: new string[0],
+    references: new[] { "System.Core.dll"
+    options: "-infer autopropertiesensures -suggest requires -includesuggestionsinregression -premode combined -suggest requires -nonnull -bounds -show unreached -show progress -check assumptions -infer methodensures -suggest asserttocontracts"
+    contractReferenceAssemblies: true"
+    exe: false"
+     ),
+  new Options(
+    sourceFile: @"Microsoft.Research\RegressionTest\Inference\AssertToContracts.cs"
+    compilerOptions: "/optimize /define:AGGRESSIVEINFERENCE"
+    libPaths: new string[0],
+    references: new[] { "System.Core.dll"
+    options: "-infer autopropertiesensures -suggest requires -includesuggestionsinregression -premode combined -suggest requires -nonnull 
```

Comment:
:memo: It didn't work to return the `Options` instances directly from this member data property. It may have something to do with the way test discovery and executions run in separate AppDomains. Rather than deal with the underlying cause, I found that simply returning and index was a very easy solution that is sure to work.